### PR TITLE
RELEASE: Subscriptions

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

File: apps/api/src/hyp3_api/api-spec/job_parameters.yml.j2 [hunk 1]
```diff
@@ -3,34 +3,50 @@ components:
     job_type:
       description: Type of process to run this job.
       type: string
-      example: {{ job_types.values() | map(attribute='api_job_type') | first }}
+      example: RTC_GAMMA
```
Comment (Jlrine2) [inline apps/api/src/hyp3_api/api-spec/job_parameters.yml.j2:6]:
```suggestion
      example: {{ job_types.values() | map(attribute='api_job_type') | first }}
```

File: apps/main-cf.yml [hunk 2]
```diff
@@ -224,3 +236,23 @@ Resources:
...
+      KeySchema:
+        - AttributeName: subscription_id
+          KeyType: HASH
+      GlobalSecondaryIndexes:
+        - IndexName: user_id
+          KeySchema:
+            - AttributeName: user_id
+              KeyType: HASH
+          Projection:
+            ProjectionType: ALL
```
Comment (jhkennedy) [inline apps/main-cf.yml:258]: trailing new line?

File: apps/render_cf.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def render_template(template_file, job_types, env):
 def render_templates(job_types):
     env = get_env()
     for template_file in Path('.').glob('**/*.j2'):
-        render_template(template_file, job_types, env)
+        render_template(PurePosixPath(template_file), job_types, env)
```
Comment (jhkennedy) [inline apps/render_cf.py:20]: Why a PurePosixPath here?

**Review by jhkennedy — APPROVED** [review]
This looks good to me. just one nit-type suggestion, and a question.

**Pull request merged**